lib/private/User/Session.php

```diff
@@ -878,25 +888,38 @@ public function loginWithCookie($uid, $currentToken, $oldSessionId) {
 			$this->logger->info('Tried to log in {uid} but could not verify token', [
 				'app' => 'core',
 				'uid' => $uid,
+				'user' => $uid,
```

Comment:
We log `$uid` twice. Should it be `$user`?
```suggestion
				'user' => $user,
```
Later in code, we have other cases like `'user' => $uid` or `'uid' => $user`

Reply:
`$user` is an `IUser` which can not be written to the log.

The thing here is that `user` is a magic array key so he wants to set that, while not touching `uid` which is used in the log message itself.

Reply:
Thanks for the explanation!